frontend/src/views/Simulator/SettingsView.vue

```diff
@@ -408,5 +452,29 @@ const handleCreateNewValidator = async () => {
         </button>
       </div>
     </Modal>
+    <Modal :open="resetStorageModalOpen" @close="closeResetStorageModal">
+      <div class="flex flex-col">
+        <div class="flex justify-between">
+          <div class="text-xl">Reset Simulator Storage</div>
+        </div>
+        <div class="flex justify-between font-bold bg-slate-100 p-2 mt-4">
+          Are you sure you want to reset the simulator storage?
+        </div>
+        <div class="flex flex-col p-2 mt-2">
+
+          <div class="py-2 w-full">
+            All the contracts examples that are not modified will be set to their initial state.
```

Comment:
@bradleySuira, as we discussed, the modified examples are going to be also reset so we need to specify that as well

Reply:
fixed